lib/btc-utils.js

```diff
@@ -108,7 +107,7 @@ const waitForBitcoinTxToBeInMempool = async (btcTxHelper, btcTxHash, maxAttempts
       const isTxInMempool = bitcoinMempool.includes(btcTxHash);
       if(!isTxInMempool) {
         logger.debug(`[${waitForBitcoinTxToBeInMempool.name}::${bitcoinMempoolHasTx.name}] Attempting to check if the btc tx (${btcTxHash}) was already mined since it's not in the mempool yet.`);
-        const tx = await btcTransactionHelper.getTransaction(btcTxHash);
+        const tx = await btcTxHelper.getTransaction(btcTxHash);
```

Comment:
Bug 1 in this `waitForBitcoinTxToBeInMempool` function. It didn't have an `await`.